keyboards/handwired/aranck/readme.md

```diff
@@ -0,0 +1,23 @@
+# Aranck
+
+![aranck](https://i.imgur.com/IWfovI2l.jpg)
+
+![aranck2](https://i.imgur.com/rrhEhQZl.jpg)
+
+A handwired 40% ortholinear keyboard.
+
+The story: https://imgur.com/a/4QI7ifJ
+
+Keyboard Maintainer: [Arda Kilicdagi](https://github.com/ardakilic)
+
+Hardware Supported: Pro Micro
```

Comment:
If you end a line with two spaces, GitHub makes the next line feed character a single line break, which is how we like this section formatted.

My fault on the previous review; I didn't take into account how the previous line can affect subsequent lines in Markdown.

If you replace Lines 11-13 with the following, it'll be how we like.

```Markdown
Keyboard Maintainer: [Arda Kilicdagi](https://github.com/ardakilic)  
Hardware Supported: Pro Micro
```

Reply:
Should be resolved in https://github.com/qmk/qmk_firmware/pull/7151/commits/9b36328dbe9ed383dd98d51536c35d33990affa8 , thanks!

Reply:
@noroadsleft when I type like this in `readme.md`

```
Keyboard Maintainer: [Arda Kilicdagi](https://github.com/ardakilic)  
Hardware Supported: Pro Micro
```

It outputs the parsed markdown like:

`Keyboard Maintainer: [Arda Kilicdagi](https://github.com/ardakilic) Hardware Supported: Pro Micro`

Please see the output of the commit https://github.com/qmk/qmk_firmware/pull/7151/commits/9b36328dbe9ed383dd98d51536c35d33990affa8 here: https://github.com/Ardakilic/qmk_firmware/blob/9b36328dbe9ed383dd98d51536c35d33990affa8/keyboards/handwired/aranck/readme.md

Am I missing something? To prevent this, I used two new lines at https://github.com/qmk/qmk_firmware/pull/7151/commits/cf1238555d0690e8142f31c0ad73c445832577c2 again:  https://github.com/Ardakilic/qmk_firmware/blob/cf1238555d0690e8142f31c0ad73c445832577c2/keyboards/handwired/aranck/readme.md

Reply:
Sorry for the comment spam email, got you now, fixed this.